src/plays/react-multistep-form/ReactMultistepForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useMultistepForm } from './customHook/useMultistepForm';
+import PersonalInfo from './personalInfo';
+import ContactInfo from './contactInfo';
+import AccountInfo from './accountInfo';
+import { FormEvent, useState } from 'react';
+import './react-multistep-form.css';
+type FormData = {
+  firstName: string;
+  lastName: string;
+  address: string;
+  email: string;
+  password: string;
+};
+const INITIAL_DATA: FormData = {
+  firstName: '',
+  lastName: '',
+  address: '',
+  email: '',
+  password: ''
+};
+
+function ReactMultistepForm() {
+  const [data, setData] = useState(INITIAL_DATA);
+
+  function updateFields(fields: Partial<FormData>) {
+    setData((prev) => {
+      return { ...prev, ...fields };
+    });
+  }
+  const { isLastStep, isFirstStep, currentStepIndex, step, next, back, totalSteps } =
+    useMultistepForm([
+      <PersonalInfo {...data} updateField={updateFields} />,
+      <ContactInfo {...data} updateField={updateFields} />,
+      <AccountInfo {...data} updateField={updateFields} />
+    ]);
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    if (!isLastStep) return next();
+
+    alert(JSON.stringify(data));
```

Comment:
Instead of alert, can we show the output in a styled DIV below the form?

Reply:
Sure. I will work on it. Thank you.